includes/classes/REST/Tokens.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Jobber REST API for Token Management.
+ *
+ * @package Jobber
+ */
+
+namespace Jobber\REST;
+
+use WP_Error;
+use WP_REST_Server;
+use WP_REST_Request;
+use WP_REST_Response;
+
+/**
+ * Jobber API Token Management
+ */
+class Tokens extends API {
+
+	/**
+	 * API Route
+	 *
+	 * @var string
+	 */
+	protected static $route = '/token';
+
+	/**
+	 * Hook module into WP.
+	 *
+	 * @return void
+	 */
+	public function register() {
+		add_action( 'rest_api_init', [ $this, 'register_routes' ] );
+	}
+
+	/**
+	 * Register the REST API routes.
+	 *
+	 * @return void
+	 */
+	public function register_routes() {
+		register_rest_route(
+			self::$namespace,
+			self::$route,
+			[
+				'methods'  => WP_REST_Server::CREATABLE,
+				'callback' => [ $this, 'save_tokens' ],
```

Comment:
I think we need to add the `permission_callback` here, otherwise a notice will be logged